src/screens/Home/Home.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { BtnAdd, TextBtn, CryptoList } from './styles';
+import { SafeAreaView } from 'react-native';
+import { useAppSelector } from '../../Hooks/hooksState';
+import ViewCrypto from '../../components/ViewCrypto';
+import { RootState } from '../../store';
+import { Crypto } from '../../interface';
+
+const Home = (): JSX.Element => {
+  const dataCryptos: Crypto[] = useAppSelector(
+    (state: RootState) => state.cryptos.cryptos,
+  );
+  return (
+    <SafeAreaView>
+      <CryptoList
+        data={dataCryptos}
+        keyExtractor={(item: Crypto) => item.id}
```

Comment:
unnecessary type declaration, FlatList already know what type is it. and use destructuring

keyExtractor={({id}) => id}